.prettierignore

```diff
@@ -3,6 +3,7 @@ node_modules
 /build
 /.svelte-kit
 /package
+/.github
```

Comment:
Why would you ignore it in Prettier? Prettier can format YAML files, and this folder does not contain any auto-generated files.